documentation/CP-plugins-documentation/supported_plugins.md

```diff
@@ -22,6 +22,7 @@ Those plugins that do have extra documentation contain links below.
 | PixelShuffle          | PixelShuffle takes the intensity of each pixel in an image and randomly shuffles its position. | No | | N/A |
 | Predict               | Predict allows you to use an ilastik pixel classifier to generate a probability image. CellProfiler supports two types of ilastik projects: Pixel Classification and Autocontext (2-stage). | No |  | N/A |
```

Comment:
Un-support this one (but please do add the info about Pixel Classification and Autocontext)